source/docs/articles/sites/deploy-log.md

```diff
@@ -0,0 +1,46 @@
+---
+title: The Pantheon Deploy Log

```

Comment:
"Deploying Code from Dev to Test to Live" (goal oriented titles > tool oriented titles)